pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -517,11 +523,11 @@ public static Map<String, List<KitRequestShipping>> getKitRequests(@NonNull DDPI
     }
 
     private static Map<String, List<KitRequestShipping>> getKitRequests(@NonNull String realm, String target, String kitType,
-                                                                        boolean getAll) {
+                                                                        Integer kitTypeId, boolean getAll) {
```

Comment:
Perhaps the `kitType` parameter should be name `kitTypeName`, now that we have a `kitTypeId`?